CairoMakie/src/utils.jl

```diff
@@ -595,7 +595,7 @@ function per_face_colors(_color, matcap, faces, normals, uv)
     elseif color isa Colorant
         return FaceIterator{:Const}(color, faces)
     elseif color isa AbstractVector{<: Colorant}
-        return FaceIterator(color, faces)
+        return FaceIterator{:PerVert}(color, faces)
```

Comment:
Should probably point out that I changed this since it's the one thing that isn't just tests. This makes the "Colored Mesh" test work by forcing meshes to use per-vertex colors. 
Since the other backends don't do any special casing for colors that could be per face I don't really see why CairoMakie should. It also errors for 2D and 3D meshes (which is why the test was excluded) and isn't used outside of them, as far as I can tell.